crates/aptos/src/list/mod.rs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+//! A command to list resources owned by an address
+//!
+//! TODO: Examples
+//!
+
+use aptos_types::account_address::AccountAddress;
+use clap::{Parser};
+use crate::CliResult;
+use reqwest;
+
+/// Command to list resources owned by an address
+///
+#[derive(Debug, Parser)]
+pub struct ListResources {
+    /// Address of account you want to list resources for
+    account: AccountAddress,
+    #[clap(long, default_value = "https://fullnode.devnet.aptoslabs.com")]
+    node_url: String,
+}
+
+impl ListResources {
+    async fn get_resources(self) -> Result<Vec<serde_json::Value>, reqwest::Error> {
+        reqwest::get(format!("{}/accounts/{}/resources", self.node_url, self.account)).await?.json().await
+    }
+
+    // TODO: Format this in a reasonable way while providing all information
+    // add options like --tokens --nfts etc
+    fn json_string(vec: Vec<serde_json::Value>) -> String {
+        let mut output: String = "".to_owned();
+        for json in &vec {
+            output.push_str(&format!("{}\n", json.to_string()));
```

Comment:
You can drop the `to_string`, as it is redundant.  The format string `"{}"` will call to_string for you.